src/azul/plugins/repository/canned/__init__.py

```diff
@@ -134,6 +140,26 @@ def dss_deletion_subscription_query(self, prefix: str) -> JSON:
     def dss_subscription_query(self, prefix: str) -> JSON:
         return {}
 
+    def _construct_file_url(self, source_url: str, file_name: str) -> str:
```

Comment:
Is there a concern that the configured URL  for the source can be accidentally malformed and it would not get caught until the incidences are created + consumed by a downstream client. 

If so, it might be useful to try to inspect if the source is usable when the plugin is instantiated.

Reply:
I don't think so. If the URL was malformed The repository plugin should have raised an exception when `fetch_bundle` or `list_bundles` tried to use the `GitHubStagingAreaFactory` from `hca-metadata-api`.